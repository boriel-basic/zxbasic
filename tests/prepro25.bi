' Must return
' (x + y) * 2

#define test(x) x * 2

test((x + y))

