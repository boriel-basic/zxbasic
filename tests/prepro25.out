#line 1 "prepro25.bi"





(x + y) * 2

